src/core/src/sys/interactions.rs

```diff
@@ -443,4 +445,17 @@ mod test {
         interactions.add_global(Box::new(Wolf::new(1.0)));
         assert_eq!(interactions.globals().len(), 1);
     }
+
+    #[test]
+    fn test_globals_cutoff() {
```

Comment:
OK! Then I'll add a function that returns the largest cutoff. I think returning an `Option<f64>` would be the most transparent way to do it. Should I add it here or open a new PR?